### PR TITLE
SEAB-5112: Test notebook collections

### PR DESCRIPTION
**Pull request by hyunnaye**
**Description**
This PR creates integration tests that assure notebooks are supported through the collections/category endpoints.

**Review Instructions**
Verify the tests are correct and pass successfully.

**Issue**
https://ucsc-cgl.atlassian.net/browse/SEAB-5112

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that you pass the basic style checks and unit tests by running `mvn clean install`
- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] Follow the existing JPA patterns for queries, using named parameters, to avoid SQL injection
- [x] If you are changing dependencies, check the Snyk status check or the dashboard to ensure you are not introducing new high/critical vulnerabilities
- [x] Assume that inputs to the API can be malicious, and sanitize and/or check for Denial of Service type values, e.g., massive sizes
- [x] Do not serve user-uploaded binary images through the Dockstore API
- [x] Ensure that endpoints that only allow privileged access enforce that with the `@RolesAllowed` annotation
- [x] Do not create cookies, although this may change in the future
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.

### CODE REVIEW

**Review by svonworl — COMMENTED** [review]
Thanks for the hard work on this.  Please take a look at and around this code and determine if we need to modify it to support notebooks.  It appears to have some `case`ing that could need adjusting, and there's some related queries that possibly require a notebook analog.  Also, it might be a good idea to inspect where these queries are used, that code might need adjustment, too.  https://github.com/dockstore/dockstore/blob/e8eb3527fd1a3a49953ad983710d64579093730d/dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java#L97

**Review by coverbeck — COMMENTED** [review]
Was here, going to wait until Denis and Steve feedback addressed before further reviewing.

File: dockstore-integration-testing/src/test/java/io/dockstore/webservice/NotebookIT.java [hunk 1]
```diff
@@ -32,10 +32,7 @@
 import io.dockstore.common.MuteForSuccessfulTests;
 import io.dockstore.common.SourceControl;
 import io.dockstore.openapi.client.ApiClient;
-import io.dockstore.openapi.client.api.EntriesApi;
-import io.dockstore.openapi.client.api.OrganizationsApi;
-import io.dockstore.openapi.client.api.UsersApi;
-import io.dockstore.openapi.client.api.WorkflowsApi;
+import io.dockstore.openapi.client.api.*;
```
Comment (denis-yuen) [inline dockstore-integration-testing/src/test/java/io/dockstore/webservice/NotebookIT.java]: Avoid the wildcard 
https://wiki.oicr.on.ca/display/DOC/IDE+Settings and the link to https://github.com/dockstore/dockstore/blob/develop/codestyle.xml#L16-L17 should enforce this. 

https://www.baeldung.com/java-wildcard-imports#disadvantages-of-wildcard-imports for explanation

**Review by denis-yuen — COMMENTED** [review]
(oops, didn't realize a review wasn't requested yet)

**Review by denis-yuen — APPROVED** [review]

**Review by svonworl — APPROVED** [review]

**Review by kathy-t — APPROVED** [review]

**Pull request merged**